src/App.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import SimpleCounter from './Components/SimpleCounter';
+
+export default () => {
+  const greetings: string = 'Hello from React and TypeScript!!! - Manualy configured WebPack!!!'
```

Comment:
You don't need to set `string` type here

Reply:
@mmarkelov , Yeah, I know, freankly speaking this particular code just check that babel/preset-typescript is works as expected.
For this PR focus on webpack.config.js and package.json only.
Other code just confimation that all plugins conected via webpack is work fine.